packages/devtools_app/lib/src/bloc/views/bloc_screen.dart

```diff
@@ -0,0 +1,56 @@
+import 'package:flutter/material.dart';
+import 'package:flutter_bloc/flutter_bloc.dart';
+
+import '../../eval_on_dart_library.dart';
+import '../../globals.dart';
+import '../../screen.dart';
+import '../bloc/bloc_list_bloc.dart';
+import '../widgets/bloc_error_view.dart';
+import '../widgets/bloc_initial_view.dart';
+import '../widgets/bloc_loading_view.dart';
+import '../widgets/bloc_view.dart';
+
+class BlocScreen extends Screen {
+  const BlocScreen()
+      : super.conditional(
+            id: id,
+            requiresLibrary: 'package:flutter_bloc/',
+            title: 'Bloc',
+            icon: Icons.palette);
+
+  static const id = 'bloc';
+
+  @override
+  Widget build(BuildContext context) {
+    return const BlocScreenBody();
+  }
+}
+
+class BlocScreenBody extends StatelessWidget {
+  const BlocScreenBody({Key key}) : super(key: key);
+
+  @override
+  Widget build(BuildContext context) {
+    final evalOnDartLibrary = EvalOnDartLibrary(
+        ['package:flutter_bloc/flutter_bloc.dart'], serviceManager.service);
```

Comment:
shouldn't the library referenced be 'package:bloc/src/bloc_observer.dart'?

Reply:
yup good catch

Reply:
This was a workaround that wasn't resolved yet. Will try resolving now.